docs/body.html

```diff
@@ -121,6 +121,14 @@ <h1>Matplotlib: Visualization with Python</h1>
           <!-- optional important news -->
           <h3>News<a class="headerlink" href="#news" title="Permalink to news section">#</a></h3>
           <div class="news__item--highlight">
+            <h5 class="date"></h5>
+            <a href="https://discourse.matplotlib.org/t/successful-completion-of-gsoc/24032" class="link--offsite">Successful GSOC 2023 </a>
```

Comment:
Space should not be linked
```suggestion
            <a href="https://discourse.matplotlib.org/t/successful-completion-of-gsoc/24032" class="link--offsite">Successful GSOC 2023</a>
```